README.md

```diff
@@ -50,3 +52,10 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Create a Trello account and API key
+
+We're going to be using Trello's API to fetch and save to-do tasks. In order to call their API, you need to first create an account - https://trello.com/signup
+Then generate an API key and token by following the instructions here - https://trello.com/app-key
+Store these credentials in the `.env` file to keep secure. 
+
```

Comment:
You also need a board with the correct list names (though they are the default).

Reply:
README updated to include instructions for getting BoardID and Lists